dashboard/src/index.scss

```diff
@@ -1,38 +1,44 @@
-@import url("https://fonts.googleapis.com/css?family=Fira+Sans:300,400,700|Hind:300,400,700");
-
 body {
   margin: 0;
   padding: 0;
 }
 
-.container-fluid > .row > div {
-  min-width: 0;
-}
-
 svg.icon {
   /* Fix positioning */
   position: relative;
   bottom: -0.125em;
 }
 
-/* Override Hex underline text-decoration in buttons in alerts
-see: https://github.com/bitnami/hex/issues/216 */
-.alert .button {
-  text-decoration: none;
-}
-
 .centered-modal {
   position: absolute;
   top: 50%;
   left: 50%;
   right: auto;
   bottom: auto;
-  border: 1px solid rgb(204, 204, 204);
-  background: rgb(255, 255, 255);
+  border: 1px solid var(--cds-alias-object-border-color, #ccc);
+  background-color: var(--cds-alias-object-interaction-background), #fff;
   overflow: auto;
   border-radius: 4px;
   outline: none;
   padding: 20px;
   margin-right: -50%;
   transform: translate(-50%, -50%);
+
+  // Copied from the CdsButton style
```

Comment:
this is a workaround for an unlikely modal to match the rest of the styles:
![image](https://user-images.githubusercontent.com/11535726/108738221-af507680-7533-11eb-88d0-2af18e6267cd.png)

Reply:
what's the behavior without this `.button`?

Reply:
Something like: 
![image](https://user-images.githubusercontent.com/11535726/108832560-fc802700-75cb-11eb-9d0b-d1da99f63bb3.png)

Reply:
hum, weird, we should be missing something here but I am not sure what

Reply:
![image](https://user-images.githubusercontent.com/11535726/108840854-34409c00-75d7-11eb-9af7-bea53d4824ff.png)

I've just updated this modal to use the cds button as other components do